config/rbac/role.yaml

```diff
@@ -65,3 +65,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - watch
```

Comment:
where do we need `watch`?

Reply:
- it's probably for operator-sdk/controller-runtime to build sharedinformercache, I didn't dig into code though where that was defined
- when `watch` wasn't provided I saw errors logged in